swagger.yaml

```diff
@@ -6338,6 +6338,7 @@ definitions:
       - 'no-skip-installation-disk'
       - 'no-skip-missing-disk'
       - 'no-ip-collisions-in-network'
+      - 'no-disk-detected'
```

Comment:
why do we need a new validation? we can use has-min-valid-disks

Reply:
Yes, it already validates at least one eligible disk.  Just need to make sure that the validation appears in the UI when there are no disks at all.

Reply:
The validation is [here](https://github.com/openshift/assisted-service/blob/master/internal/host/validator.go#L1725-L1730) 

The name of the validation is a bit misleading, This is more to validate disk to format  is existing on the host 
it validate if the ID previously  found is still in inventory.  This looks like for cases of reboot / changed HW.